hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -532,25 +532,25 @@ extends Importer:
     
     // TODO extract this into a separate method
     @tailrec
-    def go(sts: Ls[Tree], annotations: Ls[Term], acc: Ls[Statement]): Ctxl[(Term.Blk, Ctx)] =
-      def reportUnusedAnnotations: Unit =
-        if annotations.nonEmpty then
-          raise(WarningReport(
-            msg"The annotation is not used" -> (annotations.foldLeft[Opt[Loc]](N):
-              case (acc, ann) => acc match
-                case N => ann.toLoc
-                case S(loc) => S(loc ++ ann.toLoc)) ::
-            (sts.headOption match
-              case N => msg"Because nothing follows" -> blk.toLoc.map(_.right)
-              case S(head) => msg"Because annotations cannot be applied to ${head.describe}" -> head.toLoc
-            ) :: Nil))
+    def go(sts: Ls[Tree], qualifiers: Ls[Term], acc: Ls[Statement]): Ctxl[(Term.Blk, Ctx)] =
```

Comment:
Why rename to annotations to qualifiers? The former is much clearer and more specific.correct. There are many forms of qualifiers, but annotations are not usually understood as qualifiers.